packages/itwinui-css/src/bridge.scss

```diff
@@ -2,95 +2,95 @@
   --iui-color-text-negative-hover: var(--iui-color-text-negative);
 
-  --iui-color-text: var(--kiwi-color-text-neutral-primary);
+  --iui-color-text: var(--ids-color-text-neutral-primary);
   --iui-color-text-hover: var(--iui-color-text);
-  --iui-color-text-accent: var(--kiwi-color-text-accent-strong);
+  --iui-color-text-accent: var(--ids-color-text-accent-strong);
   --iui-color-text-accent-hover: var(--iui-color-text-accent);
-  --iui-color-text-muted: var(--kiwi-color-text-neutral-secondary);
-  --iui-color-text-disabled: var(--kiwi-color-text-neutral-disabled);
+  --iui-color-text-muted: var(--ids-color-text-neutral-secondary);
+  --iui-color-text-disabled: var(--ids-color-text-neutral-disabled);
   --iui-color-text-disabled-hover: var(--iui-color-text-disabled);
-  --iui-color-text-placeholder: var(--kiwi-color-text-neutral-secondary);
+  --iui-color-text-placeholder: var(--ids-color-text-neutral-secondary);
 
   &[data-iui-theme='light'] {
-    --iui-color-background-hsl: 209.21deg 17.09% 97.27%; // --kiwi-color-bg-surface-primary
-    --iui-color-foreground-hsl: 214.23deg 9.08% 15.9%; // --kiwi-color-text-neutral-primary
-    --iui-color-accent-hsl: 164.5deg 68.46% 30.05%; // --kiwi-color-icon-accent-strong
-    --iui-color-informational-hsl: 210.23deg 96.74% 36.73%; // --kiwi-color-icon-info-faded
-    --iui-color-positive-hsl: 105.38deg 97.33% 21.67%; // --kiwi-color-icon-positive-faded
-    --iui-color-warning-hsl: 39.78deg 94.44% 25.8%; // --kiwi-color-icon-attention-faded
-    --iui-color-negative-hsl: 348.19deg 79.28% 40.72%; // --kiwi-color-icon-critical-faded
+    --iui-color-background-hsl: 209.21deg 17.09% 97.27%; // --ids-color-bg-page-base
+    --iui-color-foreground-hsl: 214.23deg 9.08% 15.9%; // --ids-color-text-neutral-primary
+    --iui-color-accent-hsl: 164.5deg 68.46% 30.05%; // --ids-color-icon-accent-strong
+    --iui-color-informational-hsl: 210.23deg 96.74% 36.73%; // --ids-color-icon-info-faded
+    --iui-color-positive-hsl: 105.38deg 97.33% 21.67%; // --ids-color-icon-positive-faded
+    --iui-color-warning-hsl: 39.78deg 94.44% 25.8%; // --ids-color-icon-attention-faded
+    --iui-color-negative-hsl: 348.19deg 79.28% 40.72%; // --ids-color-icon-critical-faded
   }
 
   &[data-iui-theme='dark'] {
-    --iui-color-background-hsl: 214.23deg 9.08% 15.9%; // --kiwi-color-bg-surface-primary
-    --iui-color-foreground-hsl: 209.2deg 40.21% 98.84%; // --kiwi-color-text-neutral-primary
-    --iui-color-accent-hsl: 166.21deg 96.14% 50.19%; // --kiwi-color-icon-accent-strong
-    --iui-color-informational-hsl: 210.52deg 76.91% 71.38%; // --kiwi-color-icon-info-faded
-    --iui-color-positive-hsl: 105.91deg 34.34% 61.19%; // --kiwi-color-icon-positive-faded
-    --iui-color-warning-hsl: 40.01deg 57.6% 59.24%; // --kiwi-color-icon-attention-faded
-    --iui-color-negative-hsl: 347.86deg 75.06% 76.69%; // --kiwi-color-icon-critical-faded
+    --iui-color-background-hsl: 214.23deg 9.08% 15.9%; // --ids-color-bg-page-base
```

Comment:
These hsl values need to be recalculated, because some of the colors have changed recently.